graphql_compiler/schema_generation/sqlalchemy/schema_graph_builder.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019-present Kensho Technologies, LLC.
```

Comment:
I've seen and reviewed this file in add-sqlalchemy-schema-generation feature branch. Assuming no changes.

Reply:
There are changes. The input is now different and I've removed default value parsing since we don't seem to be interested in it at the time and I would prefer to have no to maintain it.